homeassistant/components/clementine/media_player.py

```diff
@@ -67,118 +67,64 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 class ClementineDevice(MediaPlayerEntity):
     """Representation of Clementine Player."""
 
+    _attr_media_content_type = MEDIA_TYPE_MUSIC
+    _attr_supported_features = SUPPORT_CLEMENTINE
+
     def __init__(self, client, name):
         """Initialize the Clementine device."""
         self._client = client
-        self._name = name
-        self._muted = False
-        self._volume = 0.0
-        self._track_id = 0
-        self._last_track_id = 0
-        self._track_name = ""
-        self._track_artist = ""
-        self._track_album_name = ""
```

Comment:
These are initialized differently than the entity attributes, it this change accounted for? Did you test it?

Reply:
I do not own the device, so no. See last commit.

Reply:
So how do you test these changes? There aren't any unit tests on this either?

Reply:
Turns out I was able to test this PR. Clementine is a music player so I tested with my windows machine. Works perfectly even with volume and media attributes not initialized in the `__init__` block.